corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -27,7 +23,11 @@
         infoTable,
         showTable,
         listenOnTable,
-        dropTable
+        dropTable,
+        listTags,
+        listTablesForTag,
+        listTagsForTable,
+        listTagsMap
```

Comment:
nit - is it better to call this listTagToTableMap or something similar?

Reply:
I had it like that, just thought it was too long... no particular preference, thought actually that people would complain for it as it is expected to be one of the most used operations...